tests/utils.py

```diff
@@ -106,6 +106,11 @@ def tearDown(self):
         np.testing.assert_array_almost_equal(self._x_test_iris_original, self.x_test_iris, decimal=3)
         np.testing.assert_array_almost_equal(self._y_test_iris_original, self.y_test_iris, decimal=3)
 
+    def assertEqual(self, first, second, msg=None):
+        first = first.tolist() if isinstance(first, np.ndarray) else first
+        second = second.tolist() if isinstance(second, np.ndarray) else second
+        super(TestBase, self).assertEqual(first, second, msg)
+        
```

Comment:
Did you eventually use this function somewhere?

Reply:
Oh, is this overwriting the `assertEqual` of unittest? I'm not sure of this might not have unexpected consequences. Would it be possible to use `numpy.testing.assert_equal` for testing equality of numpy arrays?

Reply:
Hi, thanks for your review. I have just tested the function of `numpy.testing.assert_equal` and it should work for the puprose. I will make a change to them.